packages/@sanity/desk-tool/src/pane/EmptyListPane.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import DefaultPane from 'part:@sanity/components/panes/default'
+import styles from './styles/EmptyListPane.css'
+
+export default function EmptyListPane(props) {
+  return (
+    <DefaultPane title="Empty schema" isScrollable={false} staticContent="&nbsp;">
```

Comment:
The &nbsp; here is a hack to make DefaultPane render its children. Please help @kristofferj ?